java/src/main/resources/template.properties

```diff
@@ -207,4 +207,19 @@ kafka.bq.await.termination.timeout=420000
 ## Fails the job when data is lost. Accepted values: true, false
 kafka.bq.fail.on.dataloss=false
 ## Ouptut mode for writing data. Accepted values: 'append', 'complete', 'update'
-kafka.bq.stream.output.mode=append
\ No newline at end of file
+kafka.bq.stream.output.mode=append
+
+
+
+#Kafka Common
+kafka.message.format=
+kafka.bootstrap.servers=
+kafka.topic=
+kafka.starting.offset=
+kafka.schema.url=
+
+#Kafka To GCS
+kafka.gcs.output.location=;
+kafka.gcs.output.format=parquet
+kafka.gcs.output.mode=append
+kafka.gcs.await.termination.timeout=600000
```

Comment:
what is unit of measurement here.
Please rename the property to reflect the same like kafka.gcs.await.termination.timeout.msec / .sec / .min etc

Reply:
Appended ".ms" to all instances of termination timeout to indicate time unit